### DISCUSSION TITLE
Just a very simple Thank you Nick.

### DISCUSSION THREAD

**Discussion by LeeWheeler** [nickcoutsos/keymap-editor#153] [General]
I've been editing my keymap by hand fore the last few years (Dactyl Manuforms 6 x 6) spent many an hour trying to figure out why I just broken the Keymap.  Many thanks for your time and effort Nick.

**Comment by nickcoutsos** [comment]
Glad to hear it's been helpful! I'd be interested in knowing how well that's working out for you, as Dactyl's are varied enough that I can't tailor make layout's to match the physical keyboard. Is the auto-generated layout good enough in practice?

**Comment by LeeWheeler** [comment]
Hi Nick, The 6x6 Dactyl”s that I use worked first time as soon as I Linked them to my git hub.I thought I was going to have to play around with changing the JSON files and set up a new controller as  My layout is 0-5 11-6 x 6 rows with a 6 key thumb cluster.  But it worked 100%I’ve had no issues with my layout, your interface is easy and efficient,  and I’m really loving it.I dont use macros or fancy things as mine is mostly for gaming and programming but I do have all of my keys mapped over 3 layers so working great.Your time invested in this project is greatly appreciated and I’m thankful for all the time you will save me.Again many thanksSent from my iPhone XS Max 512GBOn 12 Nov 2023, at 19:36, Nick Coutsos ***@***.***> wrote:﻿
Glad to hear it's been helpful! I'd be interested in knowing how well that's working out for you, as Dactyl's are varied enough that I can't tailor make layout's to match the physical keyboard. Is the auto-generated layout good enough in practice?
